src/views/Guide.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <v-container>
+    <h1 style="margin:0;">Guide</h1>
```

Comment:
The margins from `h1`, `h2`, etc, are annoying at times. We can instead use `span` or `p`, and annotate with some typography class like `display-1` and `font-weight-thin`, etc.

So this gives similar result:

```
<p class="display-3 font-weight-thin mb-3">Guide</p>
```

(`display-3` is larger than `display-1`, there are multiple `font-weight-` classes, and `mb` stands for `margin-bottom`, you can also use `mx-`, `my-`, `mt-`, ditto for padding with `px` etc)

Reply:
So yes you can customise a `<p />` to behave as you would like `<hx />` to work, however, it's just as easy to customise `<hx />` to behave the way you want it to in the first place.

The big issue with using `<p />` or `<span />` for headings is that is is a problem for accessibility.

Reply:
Oh good point. In that case, maybe this then?

`<h1 class='ma-0'>` 

`ma-0` is the class for no margins.

Reply:
Ah, very nice!